CHANGELOG.md

```diff
@@ -2,42 +2,44 @@
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## Platform Version 0.11.13-dev - UNRELEASED
+
 ## Platform Version 0.11.12-dev - 2024-09-13
```

Comment:
We missed in in the 0.11.12 release, it  should be 0.11.12 with no "-dev"

Reply:
Also fixed!